third_party/2and3/redis/client.pyi

```diff
@@ -65,9 +65,14 @@ class Redis(object):
         ssl: bool = ...,
         ssl_keyfile: Optional[Text] = ...,
         ssl_certfile: Optional[Text] = ...,
-        ssl_cert_reqs: Optional[Union[str, int]] = ...,
+        ssl_cert_reqs: Union[str, int] = ...,
         ssl_ca_certs: Optional[Text] = ...,
+        ssl_check_hostname: bool = ...,
         max_connections: Optional[int] = ...,
+        single_connection_client: bool = ...,
+        health_check_interval: int = ...,
```

Comment:
As this gets added to the result of `time()`, I expect that it also accepts `float` values:

```suggestion
        health_check_interval: float = ...,
```

Reply:
updated!